src/components/molecules/NavbarSchedule/NavbarSchedule.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback, useState } from "react";
+import classNames from "classnames";
+
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+
+import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
+
+import "./NavbarSchedule.scss";
+
+export const NavbarSchedule: React.FC = () => {
+  const { sovereignVenueId: navbarTitle } = useSovereignVenueId();
+
+  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
+
+  const toggleEventSchedule = useCallback(() => {
+    setEventScheduleVisible((scheduleVisible) => !scheduleVisible);
+  }, []);
+
+  const hideEventSchedule = useCallback(() => {
+    setEventScheduleVisible(false);
+  }, []);
```

Comment:
You can use `src/hooks/useShowHide.ts` for this:

```suggestion
const {
  isShown: isEventScheduleVisible,
  toggle: toggleEventSchedule,
  hide: hideEventSchedule,
} = useShowHide(false);
```